packages/common/ecs.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const aws = require('./aws');
+
+/**
+ * list the clusters
+ *
+ * @returns {Array<string>} the cluster arns
+ */
+async function listEcsClusters() {
+  let response;
+  let clusterArns = [];
+  do {
+    const nextToken = response ? response.nextToken : null;
+    const params = { maxResults: 100, nextToken };
+    // eslint-disable-next-line no-await-in-loop
+    response = await aws.ecs().listClusters().promise(params);
```

Comment:
`params` should be an argument of `listClusters()`, not `promise()`. As written, this would cause an infinite loop if nextToken was ever returned by AWS.

```suggestion
    response = await aws.ecs().listClusters(params).promise();
```